packages/next/server/render.tsx

```diff
@@ -149,6 +151,7 @@ class ServerRouter implements NextRouter {
     this.domainLocales = domainLocales
     this.isPreview = !!isPreview
     this.isLocaleDomain = !!isLocaleDomain
+    this.key = createKey()
```

Comment:
We shouldn't need to have a unique key during SSR as the key is meant to be used on the client only right?

Reply:
Right.
Should it be `null` in SSR?

Reply:
I think leaving this unset during SSR would be fine and doesn't need to be set anyways if it's not exposed publicly.